include/onnxruntime/core/framework/tensor.h

```diff
@@ -58,15 +58,18 @@ using BufferNakedPtr = void*;
 class Tensor final {
  public:
   /**
-     Create tensor with given type, shape, pre-allocate memory and allocator info.
-  */
-  Tensor(MLDataType p_type,
-         const TensorShape& shape,
-         BufferNakedPtr p_data,
-         const OrtAllocatorInfo& alloc,
-         AllocatorPtr deleter = nullptr,
+   * Create tensor with given type, shape, pre-allocate memory and allocator info.
+   * This function won't check if the preallocated buffer(p_data) has enough room for the shape.
+   */
+  Tensor(MLDataType p_type, const TensorShape& shape, BufferNakedPtr p_data, const OrtAllocatorInfo& alloc,
```

Comment:
>BufferNakedPtr [](start = 54, length = 14)

Why not just use void*? BufferNakedPtr seems an unnecessary obfuscation.

Reply:
I don't know. I didn't write this code.